CMakeLists.txt

```diff
@@ -21,27 +21,27 @@ find_package( ecbuild 3.5 REQUIRED HINTS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CUR
 include( ecbuild_bundle )
 ecbuild_bundle_initialize()
 
-ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda-internal/jedi-cmake.git" BRANCH develop UPDATE )
+ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/JCSDA/jedi-cmake.git" BRANCH develop UPDATE RECURSIVE)
```

Comment:
Note for the future. You can use jedi-cmake from spack-stack if it's available, or fall back to building it here. See: https://github.com/JCSDA/jedi-bundle/blob/e6037876c15e7cf0236cb6d41c83b59a18239ad8/CMakeLists.txt#L24